hs_modflow_modelinstance/models.py

```diff
@@ -0,0 +1,799 @@
+from lxml import etree
+
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db import models
+from django.core.exceptions import ValidationError, ObjectDoesNotExist
+
+from mezzanine.pages.page_processors import processor_for
+
+import os
+
+from hs_core.models import BaseResource, ResourceManager, resource_processor, \
+    AbstractMetaDataElement
+
+from hs_modelinstance.models import ModelInstanceMetaData, ModelOutput, ExecutedBy
+
+
+def delete_if_empty(term_obj):

```

Comment:
I think it would be better to pass a 2nd parameter that is a list of attribute names to be checked for the 1st parameter ```term_obj`` instead of using the standard_elements.